doc/getting_started/installation_win.rst

```diff
@@ -20,6 +20,145 @@ Windows system with the latest updates installed.
+
+#. Install :program:`Chocolatey` by following the instructions on the
+   `Chocolatey install`_ website.
+
+#. Open a Command Prompt (`cmd.exe`) as an **Administrator**.
+
+#. Optionally disable global confirmation to avoid having to add `-y` to all
+   commands:
+
+   .. code-block:: console
+
+      > choco feature enable -n allowGlobalConfirmation
+
+#. Install CMake and DTC:
+
+   .. code-block:: console
+
+      > choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+      > choco install dtc-msys2 --version 1.4.4
+
+#. Install the rest of the tools:
+
+   .. code-block:: console
+
+      > choco install git python ninja gperf
+
+#. Close the Command Prompt window.
+
+#. Open a Command Prompt (`cmd.exe`) as a **regular user**.
+
+#. Clone a copy of the Zephyr source into your home directory using Git.
+
+   .. code-block:: console
+
+      > cd %userprofile%
+      > git clone https://github.com/zephyrproject-rtos/zephyr.git
+
+#. Install the required Python modules::
+
+      > cd %userprofile%\zephyr
+      > pip install --user -r scripts/requirements.txt
+
+#. The build system should now be ready to work with any toolchain installed in
+   your system. In the next step you'll find instructions for installing
+   toolchains for building both x86 and ARM applications.
+
+#. Install cross compiler toolchain:
+
+   * For x86, install the 2017 Windows host ISSM toolchain from the Intel
+     Developer Zone: `ISSM Toolchain`_. Use your web browser to
+     download the toolchain's ``tar.gz`` file. You can then use 7-Zip or a
+     similar tool to extract it into a destination folder.
+
+     .. note::
+
+        The ISSM toolset only supports development for Intel |reg| Quark |trade|
+        Microcontrollers, for example, the Arduino 101 board.  (Check out the
+        "Zephyr Development Environment
+        Setup" in this `Getting Started on Arduino 101 with ISSM`_ document.)
+        Additional setup is required to use the ISSM GUI for development.
+
+
+   * For ARM, install GNU ARM Embedded from the ARM developer website:
+     `GNU ARM Embedded`_ (install to :file:`c:\\gccarmemb`).
+
+#. Within the Command Prompt, set up environment variables for the installed
+   tools and for the Zephyr environment:
+
+   For x86:
+
+   .. code-block:: console
+
+      > set ZEPHYR_GCC_VARIANT=issm
+      > set ISSM_INSTALLATION_PATH=c:\issm0-toolchain-windows-2017-01-25
+
+   Use the path where you extracted the ISSM toolchain.
+
+   For ARM:
+
+   .. code-block:: console
+
+      > set ZEPHYR_GCC_VARIANT=gccarmemb
+      > set GCCARMEMB_TOOLCHAIN_PATH=c:\gccarmemb
+
+   And for either, set the `ZEPHYR_BASE` environment variable to the root
+   of the Zephyr repository you cloned:
+
+   .. code-block:: console
+
+      > set ZEPHYR_BASE=%userprofile%\zephyr
+
+#. Finally, you can try building the :ref:`hello_world` sample to check things
+   out.
+
+To build for the Intel |reg| Quark |trade| (x86-based) Arduino 101:
```

Comment:
This block of text (lines 143-157) should be indented with the content of this step so it stays as part of the step.